apps/judicial-system/api/src/app/modules/indictment-count/models/indictmentCount.model.ts

```diff
@@ -36,9 +32,6 @@ export class IndictmentCount {
   @Field(() => [Offense], { nullable: true })
   readonly offenses?: Offense[]
 
-  @Field(() => [IndictmentCountOffense], { nullable: true })
-  readonly deprecatedOffenses?: IndictmentCountOffense[]
-
   @Field(() => GraphQLJSONObject, { nullable: true })
   readonly substances?: SubstanceMap
```

Comment:
We can also remove the substances

Reply:
I removed all traces of substances in indictment count and cleaned up some other type stuff related to it.